atlascope/core/migrations/0002_base_models.py

```diff
@@ -24,8 +22,8 @@ class Migration(migrations.Migration):
             fields=[
                 (
                     'id',
-                    models.UUIDField(
-                        default=uuid.uuid4, editable=False, primary_key=True, serialize=False
+                    models.AutoField(
+                        auto_created=True, primary_key=True, serialize=False, verbose_name='ID'
```

Comment:
Did you choose to include this verbose name argument or was that generated by makemigrations? I'm concerned that the all-caps version may violate some linting rules if we reference this field that way.

Reply:
I am pretty sure that came from `makemigrations`. I probably didn't run `makemigrations` myself here, and instead lifted the `AutoField` line from another Girder4 project's 0002 migration. I think I also created a random model in the codebase just to see what popped out of `makemigrations`, and maybe that's where I lifted this from. Sorry, I can't quite remember what I did, but if you're worried about it you can run your own such test to see what it spits out.

Reply:
If you lifted the `AutoField` line from somewhere else, it's probably safer to take out the `verbose_name` argument from these lines.

Reply:
So, I did the following as a test: in the jobs model file, I created a new model class and added a foreign key field to the Job model pointing to it (so that makemigrations would pick it up). The generated migration has the verbose name bit in it, so I am thinking this is probably fine as is. What do you think?

Reply:
Ok that should be fine, then. I doubt we will use the uppercase version